stan/functions/ct.stan

```diff
@@ -36,7 +43,7 @@ vector[] rel_inf_prob(vector prob_inf, vector ldtp, int ctmax, int t) {
     for (j in 1:ctmax) {
       lrit[p][j] += prob_inf[i - j];
     } 
-   lrit[p] = log(lrit[p]) + ldtp;
+   lrit[p] = log(lrit[p]);
```

Comment:
so I am not clear how the changes above (aside from the reversing) are different from this which was I thought working out probability of infection given detection (which is the measure derived in the hay code). 

If the reverse switch wasn't there this would be the same thing. I'm open to changing the direction (I mean whatever works) but I have not justification as to why it would be a good idea.